.gitignore

```diff
@@ -11,6 +11,12 @@ neon.iml
 /integration_tests/.neon
 compaction-suite-results.*
 
+# sql_exporter config files generated from Jsonnet
+compute/etc/neon_collector.yml
+compute/etc/neon_collector_autoscaling.yml
+compute/etc/sql_exporter.yml
+compute/etc/sql_exporter_autoscaling.yml
```

Comment:
Optional: we can move this part to a `.gitignore` in `compute/` or `compute/etc/` (these lines look a bit too specific for global `.gitignore`, but that's not really a big deal, so I'm ok to leave it as is).